packages/bsdowl/bsdowl.3.0.0-20150401/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "michipili@gmail.com"
+authors: "Michael Grünewald"
+license: "CeCILL-B"
+homepage: "https://github.com/michipili/bsdowl"
+bug-reports: "https://github.com/michipili/bsdowl/issues"
+dev-repo: "https://github.com/michipili/bsdowl.git"
+tags: [
+  "bsd"
+  "bmake"
+  "build"
+]
+build: [
+  ["./configure" "--prefix" prefix]
+  ["bmake" "-r" "build"] {os != "freebsd"}
```

Comment:
@samoht What do you mean? If I just say `make` (the opam variable) I will probably run GNU Make. but I want `bmake` which is available as `bmake` on most platforms but as `make` under FreeBSD.